lib/Dialect/Calyx/Export/CalyxEmitter.cpp

```diff
@@ -1044,14 +1067,20 @@ void Emitter::emitLibraryFloatingPoint(Operation *op) {
   }
 
   indent() << getAttributes(op, /*atFormat=*/true) << cell.instanceName()
-           << space() << equals() << space() << opName << LParen() << expWidth
-           << comma() << sigWidth << comma() << bitWidth;
+           << space() << equals() << space() << opName << LParen();
+
+  if (auto intToFpOp = dyn_cast<calyx::IntToFpOpIEEE754>(op)) {
+    unsigned intWidth = getIntWidth(op, cell, /*isIntToFp=*/true);
```

Comment:
Nit: instead of passing a `bool` argument, just do `isa<calyx::IntToFpOpIEEE754>` the function.